### DISCUSSION TITLE
How do I get prettier to ignore adding new lines in inner text

### DISCUSSION THREAD

**Discussion by nurdism** [prettier/prettier#12654] [Q&A] [answered]
Not sure how to explain: 

Prettier is doing this: 
![](https://i.imgur.com/opve3Xz.png)

I want it to do this: 
![](https://i.imgur.com/awKzJMr.png)

Can I change this behavior?

My `.prettierrc`:
```json
{
  "semi": false,
  "arrowParens": "always",
  "trailingComma": "all",
  "singleQuote": true,
  "printWidth": 180,
  "tabWidth": 2,
  "bracketSameLine": true,
  "vueIndentScriptAndStyle": true,
  "htmlWhitespaceSensitivity": "ignore",
  "endOfLine": "lf",
  "braceStyle": "1tbs"
}
```

**Comment by cerm88** [comment, marked as answer]
I think that already depends on the "printWidth" property, play with it by lowering the number of characters
